src/Workspaces/Core/Portable/FindSymbols/SymbolFinder.cs

```diff
@@ -271,7 +272,15 @@ public static IEnumerable<TSymbol> FindSimilarSymbols<TSymbol>(TSymbol symbol, C
                 foreach (var linkedDocumentId in originalDocument.GetLinkedDocumentIds())
                 {
                     var linkedDocument = solution.GetRequiredDocument(linkedDocumentId);
+
+                    // It's possible for us to have a solution snapshot where only part of a linked set of documents has
```

Comment:
>b [](http://example.com/codeflow?start=33&length=1)

I'm wondering if we need to handle this in completion as well. Here for example: [https://sourceroslyn.io/#Microsoft.CodeAnalysis.Features/Completion/Providers/AbstractSymbolCompletionProvider.cs,324](https://sourceroslyn.io/#Microsoft.CodeAnalysis.Features/Completion/Providers/AbstractSymbolCompletionProvider.cs,324)

Reply:
ANy place that assume the files must have the same content is definitely in error and needs to be fixed up.